Chore: Improve SecretInput story (#51665)
grafana/grafana: packages/grafana-ui/src/components/SecretInput/SecretInput.mdx

import { Props } from '@storybook/addon-docs/blocks';
import { SecretInput } from './SecretInput';

# Secret Input

> Used for secret/password input it has 2 states **_configured_** and **_not configured_**.

Reviewer: ```suggestion
Used for secret/password input. It has 2 states: **_configured_** and **_not configured_**.
```

> - If configured it will not show its contents and adds a reset button that will
>   clear the input and makes it accessible.

Reviewer: ```suggestion
- If configured it will disable the input and add a reset button that will
  clear the input and make it accessible.
```

> - In non configured state it behaves like normal text input.

Reviewer: ```suggestion
- In non configured state it behaves like a normal password input.
```
